.eslintrc

```diff
@@ -81,7 +81,7 @@
     "semi": [2, "always"],
     "space-before-blocks": [2, "always"],
     "space-before-function-paren": [2, "never"],
-    "space-infix-ops": 0,
+    "space-infix-ops": 2,
```

Comment:
oh, ha, I forgot we even added it. I guess it was because we liked doing default assignment in Python style. I realize we need to enforce it for things like `if (thing===other)` which is more important though.